libs/core/src/lib/carousel/carousel.component.ts

```diff
@@ -249,6 +251,7 @@ export class CarouselComponent implements OnInit, AfterContentInit, AfterViewIni
             this.navigation = false;
         }
 
+        this.slides.changes.subscribe(() => this._onSlideUpdates());
```

Comment:
You need to close this subscription `onDestroy` of this component. Add `.pipe(takeUntil(this._onDestroy$)).subscribe(...)` as you have done elsewhere in this file. Also I noticed other subscriptions in this file where you need to do this - go through and make sure you're closing any subscriptions you have opened.

Reply:
this is fixed now. using pipe(takeUntil(this._onDestroy$)) now.